airflow/providers/google/cloud/transfers/local_to_gcs.py

```diff
@@ -109,6 +109,8 @@ def execute(self, context: 'Context'):
         )
 
         filepaths = self.src if isinstance(self.src, list) else glob(self.src)
+        if len(filepaths) == 0:
+            raise ValueError(f"src {self.src} dos not exist.")
```

Comment:
```suggestion
        if not filepaths:
            raise FileNotFoundError(self.src)
```

Reply:
Updated with the above change 42cbdb734990e8d7f8cebf987791c552b384b951